src/components/PopoverMenu/index.js

```diff
@@ -41,14 +41,14 @@ const defaultProps = {
 function PopoverMenu(props) {
     const {isSmallScreenWidth} = useWindowDimensions();
     const [selectedItemIndex, setSelectedItemIndex] = useState(null);
+    const [focusedIndex, setFocusedIndex] = useArrowKeyFocusManager({initialFocusedIndex: -1, maxIndex: props.menuItems.length - 1, isActive: props.isVisible});
 
     const selectItem = (index) => {
         const selectedItem = props.menuItems[index];
         props.onItemSelected(selectedItem);
         setSelectedItemIndex(index);
     };
 
-    const [focusedIndex, setFocusedIndex] = useArrowKeyFocusManager({initialFocusedIndex: -1, maxIndex: props.menuItems.length - 1});
     useKeyboardShortcut(
```

Comment:
Since you are recommending to wrap the callback function with `useCallback` when using `useKeyboardShortcut`, you could apply here as well.

Reply:
In this case there is no need given the callback will use `focusedIndex` which updates whenever the key UP/DOWN is press, also the `selectItem` uses the prop `onItemSelected` which isn't stable, we will need to go up in the tree and fix with `useCallback` or memo when possible the parent components